.circleci/config.yml

```
@@ -187,6 +187,46 @@ jobs:
 
               CI_COMMIT_BRANCH="master" python3 ./Tests/scripts/update_conf_json.py
               CI_COMMIT_BRANCH="master" ./Tests/scripts/validate.sh
+  test-lint:
+      docker:
+        - image: devdemisto/content-build:3.0.0.12681
+      steps:
+        - checkout
+        - setup_remote_docker
+        - attach_workspace:
+            at: ~/project
+        - run:
+            name: Test lint on python and ps files
+            when: always
+            command: |
+              cd ~/project
+              export SDK_LINT_FILES_CHANGED=$(git diff master... --name-only -- demisto_sdk/commands/lint)
+
+              if [[ -n "${SDK_LINT_FILES_CHANGED}" ]]; then
```

it will be easier to read if we switch the condition.
```suggestion
              if [[ -z "${SDK_LINT_FILES_CHANGED}" ]]; then
                  echo "files under demisto_sdk/commands/lint did not change - Skipping lint runs."
                  exit 0
              fi
```